detect_variants/detect_variants.cwl

```diff
@@ -102,10 +105,39 @@ steps:
             vcf: combine/combined_vcf
         out:
             [filtered_vcf]
+    bgzip:
+        run: bgzip.cwl
+        in:
+            file: filter/filtered_vcf
+        out:
+            [bgzipped_file]
+    index:
```

Comment:
There are already steps in `detect_variants.cwl` named `bgzip` and `index`.  They'll need to be unique or else references like `bgzip/bgzipped_file` become ambiguous.